### PR TITLE
Fix closure warnings with FETCH code. NFC

### PR DESCRIPTION
**Pull request by sbc100**
See #12728

### CODE REVIEW

File: src/library_fs.js [hunk 1]
```diff
@@ -1715,7 +1715,7 @@ FS.staticInit();` +
           if (datalength !== chunkSize) xhr.setRequestHeader("Range", "bytes=" + from + "-" + to);
 
           // Some hints to the browser that we want binary data.
-          if (typeof Uint8Array != 'undefined') xhr.responseType = 'arraybuffer';
+          xhr.responseType = 'arraybuffer';
```
Comment (sbc100) [inline src/library_fs.js:1718]: This is unrelated by I notices we were trying to handle older browsers where Uint8Array did not exist.. I'm pretty sure we don't support that anymore, right?

Comment (kripken) [reply src/library_fs.js:1718]: Yes, even IE11 has typed arrays. We don't even try to support things older than that. There is no efficient way to polyfill typed arrays using normal arrays in any case...
